packages/local-signing-manager/src/lib/local-signing-manager.ts

```diff
@@ -97,6 +97,7 @@ export class LocalSigningManager implements SigningManager {
   private keyring: Keyring;
   private externalSigner: KeyringSigner;
   private hasFormat?: boolean;
+  private type?: KeyRingType;
```

Comment:
```suggestion
```

Use `this.keyring.type` of introducing a new property. If you want define a private getter with this name instead.

Reply:
Sure.